source/lifecycle/mattermost-server-releases.md

```diff
@@ -0,0 +1,31 @@
+(mattermost-server-releases)=
+# Mattermost Server releases
+
+```{include} ../_static/badges/allplans-selfhosted.md
+```
+
+```{Important}
+**Support for Mattermost Server v8.1 Extended Support Release has come to the end of its life cycle on May 15, 2024. Upgrading to Mattermost Server v9.5 Extended Support Release or later is required.**
+```
+
+## Frequency
+Mattermost releases a new server version on the 16th of each month in [binary form](/upgrade/upgrading-mattermost-server). 
+- See the [Mattermost v10 changelog](/lifecycle/mattermost-v10-changelog) for details on what's coming and changing in the next major release. 
+- See the [Mattermost v9 changelog](/lifecycle/mattermost-v9-changelog) for details on v9.x server releases.
+- See the [unsupported Mattermost legacy releases](/lifecycle/unsupported-legacy-releases) documentation for details on older, unsupported  Mattermost releases.
+
+## Latest releases
+
+| **Release** | **Life cycle start** | **Life cycle end** |
```

Comment:
@amyblais - Can I get your help to review/correct the life cycle start and end dates for these server releases (based on our v9.x server changelog content) please?